src/main/java/com/dev/cinema/Main.java

```diff
@@ -64,51 +68,54 @@ public static void main(String[] args) throws AuthenticationException {
         flashSession.setShowTime(LocalDateTime.now().plusMonths(2));
         movieSessionService.add(flashSession);
 
+        log.info("Getting all Available Sessions");
         movieSessionService.findAvailableSessions(spiderMan.getId(), LocalDate.now())
-                .forEach(System.out::println);
+                .forEach(log::info);
         movieSessionService.findAvailableSessions(ironMan.getId(), LocalDate.now())
-                .forEach(System.out::println);
+                .forEach(log::info);
         movieSessionService.findAvailableSessions(flash.getId(), LocalDate.now())
-                .forEach(System.out::println);
+                .forEach(log::info);
 
         UserService userService = (UserService) injector.getInstance(UserService.class);
         User user1 = new User();
         user1.setEmail("arts@ukr.net");
         user1.setPassword("123");
         userService.add(user1);
-        System.out.println(userService.findByEmail(user1.getEmail()));
+        log.info("Find User by email " + userService.findByEmail(user1.getEmail()));
 
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
         authenticationService.register("arts2@ukr.net", "1234");
-        System.out.println(authenticationService.login("arts2@ukr.net", "1234"));
+        log.info(authenticationService.login("arts2@ukr.net", "1234"));
```

Comment:
let's wrap this in try-catch and add log.warn with the informative message in `catch` block

Reply:
done
try {
            authenticationService.login("arts2@ukr.net", "1234");
        } catch (AuthenticationException e) {
            log.info("Incorrect username or password");
        }